dataclients/kubernetes/definitions/definitions_test.go

```diff
@@ -1,11 +1,27 @@
 package definitions_test
 
 import (
+	"os"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/zalando/skipper/dataclients/kubernetes/definitions"
 	"github.com/zalando/skipper/dataclients/kubernetes/kubernetestest"
 )
 
 func TestRouteGroupValidation(t *testing.T) {
 	kubernetestest.FixturesToTest(t, "testdata/validation")
 }
+
+func TestRouteGroupsValidationErrorWrapping(t *testing.T) {
```

Comment:
This is testing ValidateRouteGroups that is not covered by TestRouteGroupValidation so it should be `TestValidateRouteGroups`. Error wrapping is a detail.